src/web-components/index.ts

```diff
@@ -0,0 +1,17 @@
+export function injectComponentScripts() {
+    injectScriptOnce(makeLearnMoreScript());
+}
+
+function injectScriptOnce(el: HTMLScriptElement) {
+    if (!document.querySelector(`[src="${el.src}"]`)) {
```

Comment:
I think it would be best to do this by adding an `id` to the tag and checking for that, but failing that...

```suggestion
    if (!document.querySelector(`script[src="${el.src}"]`)) {
```